Sprint-1/exercises/decimal.js

```diff
@@ -3,7 +3,12 @@ const num = 56.4567;
 // You should look up Math functions for this exercise https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math
 
 // Create a variable called wholeNumberPart and assign to it an expression that evaluates to 56 ( the whole number part of num )
+const wholeNumberPart = Math.floor(num);
+console.log("whole number part", wholeNumberPart);
 // Create a variable called decimalPart and assign to it an expression that evaluates to 0.4567 ( the decimal part of num )
+const decimalPart = num - wholeNumberPart;
+console.log("decimal part = ", decimalPart);
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
-
+const roundedNum = Math.ceil(num);
+console.log("rounded number = ", roundedNum);
```

Comment:
Ooh interesting! This is actually a bug in the exercise - what you've written here works for getting the right answer, but it actually doesn't quite solve the problem in the words.

I've just changed the number in https://github.com/CodeYourFuture/Module-Structuring-and-Testing-Data/pull/45 to `56.5678` - can you also change the number in your file here, and find the solution to the stated problem in words? `Math.ceil` does something slightly different from rounding to "the nearest whole number" - can you explain what it does instead?

Reply:
Ooh yeah, actually I have found out that I should have used ```Math.round``` which returns the value of a number rounded to the nearest integer instead of ```Math.ceil``` which always rounds up and returns the smallest integer greater than or equal to a given number.